handling auth tokens, code refactoring

KirillPerepelitsa opened the pull request:
Hello, 

https://github.com/lamp-io/lio/issues/1

I did code refactoring, improve token handling, updated composer.json now it has in requirements php version at least 7.0 
Token handling improvements:
1)Each command now will be injected with auth command
2)It will check, does user has stored token, on ~/.config/.lamp.io/ (i've added hack for windows, but i will be only tomorrow able to check it, does it works, dont have windows machine at home )
3) If it will be found, it just will be processed user command
4) If no, it will ask user to input it, and then it will store in a file, that i've shown above, and after it will be proceeded user command

Also i've updated,  console output with api response for already created commands, it will be shown in a table format (from my point of view it looks more user friendly)

Created 2 additional classes Console\App\Helpers\AuthHelper and Console\App\Helpers\HttpHelper, 
Auth helper will handle all token related methods, 
HttpHelper, will handle all http calls(It has been injected with GuzzleHttp\ClientInterface).

jbartus commented on src/App/Helpers/AuthHelper.php:
```diff
@@ -0,0 +1,52 @@
+	 */
+	public static function getPathToTokenFolder(): string
+	{
+		return getenv('HOME') . getenv("HOMEDRIVE") . getenv("HOMEPATH") . '/.config/.lamp.io/';
```
teeny tiny thing: should be `'/.config/lamp.io/'`.  note the `lamp.io` subdirectory is not hidden/dot-prefixed.

KirillPerepelitsa replied:
Fixed

jbartus commented:
This is great.  Small extra request: if you already have a token file, when you run `./bin/console auth` can it detect that and prompt you to keep or replace it, rather than just ignore it and overwrite it.

KirillPerepelitsa commented:
> This is great. Small extra request: if you already have a token file, when you run `./bin/console auth` can it detect that and prompt you to keep or replace it, rather than just ignore it and overwrite it.

I've updated this flow.
If token already exists user will get output,  `Token already exist, if you want to update it, please add -u/--update_token option`, so in case when he will need to update token, he just repeating command with this option, and he will be asked again to prompted his token that will overwrite current 

Also added output `Token saved` after saving token.
